lib/ansible/modules/cloud/vmware/vmware_portgroup.py

```diff
@@ -20,70 +21,78 @@
             - vSwitch to modify.
         required: True
-    portgroup_name:
+        aliases: [ 'switch_name', 'vswitch' ]
+    portgroup:
         description:
             - Portgroup name to add.
         required: True
+        aliases: [ 'portgroup_name' ]
     vlan_id:
         description:
             - VLAN ID to assign to portgroup.
-        required: True
-    network_policy:
+            - Set to 0 (no VLAN tagging) by default.
+        required: False
+        default: 0
+        aliases: [ 'vlan' ]
+    security:
         description:
             - Network policy specifies layer 2 security settings for a
               portgroup such as promiscuous mode, where guest adapter listens
               to all the packets, MAC address changes and forged transmits.
             - Dict which configures the different security values for portgroup.
             - 'Valid attributes are:'
-            - '- C(promiscuous_mode) (bool): indicates whether promiscuous mode is allowed. (default: false)'
-            - '- C(forged_transmits) (bool): indicates whether forged transmits are allowed. (default: false)'
-            - '- C(mac_changes) (bool): indicates whether mac changes are allowed. (default: false)'
+            - '- C(promiscuous_mode) (bool): indicates whether promiscuous mode is allowed. (default: None)'
+            - '- C(forged_transmits) (bool): indicates whether forged transmits are allowed. (default: None)'
+            - '- C(mac_changes) (bool): indicates whether mac changes are allowed. (default: None)'
         required: False
-        version_added: "2.2"
-        default: {
-            mac_changes: false,
-            promiscuous_mode: false,
-            forged_transmits: false,
-        }
-    teaming_policy:
+        version_added: "2.8"
+        aliases: [ 'security_policy' ]
+    teaming:
         description:
             - Dictionary which configures the different teaming values for portgroup.
             - 'Valid attributes are:'
-            - '- C(load_balance_policy) (string): Network adapter teaming policy. (default: loadbalance_srcid)'
+            - '- C(load_balancing) (string): Network adapter teaming policy. (default: loadbalance_srcid)'
             - '   - choices: [ loadbalance_ip, loadbalance_srcmac, loadbalance_srcid, failover_explicit ]'
-            - '- C(inbound_policy) (bool): Indicate whether or not the teaming policy is applied to inbound frames as well. (default: False)'
-            - '- C(notify_switches) (bool): Indicate whether or not to notify the physical switch if a link fails. (default: True)'
-            - '- C(rolling_order) (bool): Indicate whether or not to use a rolling policy when restoring links. (default: False)'
+            - '- C(notify_switches) (bool): Indicate whether or not to notify the physical switch if a link fails. (default: None)'
+            - '- C(failback) (bool): Indicate whether or not to use a failback when restoring links. (default: None)'
```

Comment:
Documentation missing for `network_failure_detection`

Reply:
done